app/views/runs/_export_button.slim

```diff
@@ -1,15 +1,16 @@
 button#export-link.btn.btn-outline-light.dropdown-toggle href='#' role='button' aria={haspopup: true, expanded: false} data={toggle: 'dropdown'}
   => icon('fas', 'download')
-  span Export
```

Comment:
I'm a bit on the fence about using `defined?`, but I don't really see a better way to do defaults for partial arguments.

Reply:
[This](https://api.rubyonrails.org/classes/ActionView/Base.html) says that local variables are stored in `local_assigns`,  so you could do something like `local_assigns.has_key?(:button_text)` for the conditional, or `local_assigns[:button_text].presence || 'Export'` as a one liner.  It should be noted that it also says using `defined?` is an alternative to check if it is defined right under that section.

Reply:
Oh sweet! I like this better.